### DISCUSSION TITLE
x509 error when adding a new sensor

### DISCUSSION THREAD

**Discussion by kiaraho** [Security-Onion-Solutions/securityonion#6997] [Unsupported Versions] [answered]
Adding a new Sensor node to an existing deployment after a recent upgrade to `2.3.91`. `so-setup-network` fails with the following error in `/root/sosetup.log` for all SO services.

```
[ERROR   ] Failed to pull so2-manager:5000/security-onion-solutions/so-wazuh:2.3.91: Error 500: Get "https://so2-manager:5000/v2/": x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
```

**Comment by kiaraho** [comment, marked as answer]
So it looks like switching the sensor OS from Ubuntu 20 to Ubuntu 18 fixed the issue. The manager is running Ubuntu 18. Maybe the difference in `openssl` versions between Ubuntu 18 and 20 has something to do with it?

**Reply by dougburks** [reply]
I've added a warning to the documentation:
https://docs.securityonion.net/en/2.3/installation.html
